services/device-registry-jdbc/src/main/java/org/eclipse/hono/deviceregistry/jdbc/impl/DeviceManagementServiceImpl.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2020, 2022 Contributors to the Eclipse Foundation
+ * Copyright (c) 2020, 2022, 2023 Contributors to the Eclipse Foundation
```

Comment:
please remove 2022 and make sure to check all your other touched files for the same issue...

Reply:
Done